packages/cli/src/ui/components/InputPrompt.tsx

```diff
@@ -50,7 +50,9 @@ export const InputPrompt: React.FC<InputPromptProps> = ({
   config,
   slashCommands,
   commandContext,
-  placeholder = '  Type your message or @path/to/file',
+  placeholder = process.platform === 'win32' 
+    ? '  Type your message or @path/to/file (Alt+V to paste image)'
+    : '  Type your message or @path/to/file',
```

Comment:
lets avoid promoting (Alt+V) here. It would be good however to promote Alt+V  or Ctrl+V in the list of keyboard shortcuts shown when you type `/help`